delatore/sources/base.py

```diff
@@ -69,16 +82,30 @@ async def start(self, stop_event: asyncio.Event):
                 new = await asyncio.wait_for(self.get_update(), self.request_timeout)
             except (asyncio.TimeoutError, NoUpdates):
                 continue
-            if _same_status(new, last) and self.ignore_duplicates:
+            if self.ignore_duplicates and \
+                    _same_status(new, last) and \
+                    _heartbeat_timestamp(new, last) >= self.heartbeat_interval:
+                json_message = json.dumps(new)
+                await self.client.publish(self.TOPICS.info, json_message)
```

Comment:
Should it be here? If not `_same_status(new, last)`, no info messagge will be published

Reply:
If not `_same_status(new, last)` it will publish in changes